src/components/UI/FacebookLogo.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+
+const FacebookLogo = () => {
+  return (
+    <Link
+      href="https://www.facebook.com/spvsp"
```

Comment:
Let's move these `href`s to an imported `constants.ts` file at root. That'll let us reuse them for SEO purposes when we get around to adding some SEO goodness via [next/head](https://nextjs.org/docs/pages/api-reference/components/head)